core/src/main/resources/reference.conf

```diff
@@ -109,6 +109,11 @@ jdbc-journal {
 
   tables {
     journal {
+      # The size of single partition and prefix of partition when we choose to use partitioned journal.
+      partitioned {
```

Comment:
We already have a config value called `partitioned` on the root level of the journal configuration. To make it less ambiguous and readable more like prose I'd suggest rename this key to `partitions`.

Reply:
changed